// numbers-from-words/dist/types.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Magnitudes = exports.Multiples = exports.Teens = exports.Numbers = void 0;
var Numbers;
(function (Numbers) {
    Numbers[Numbers["Zero"] = 0] = "Zero";
    Numbers[Numbers["One"] = 1] = "One";
    Numbers[Numbers["Two"] = 2] = "Two";
    Numbers[Numbers["Three"] = 3] = "Three";
    Numbers[Numbers["Four"] = 4] = "Four";
    Numbers[Numbers["Five"] = 5] = "Five";
    Numbers[Numbers["Six"] = 6] = "Six";
    Numbers[Numbers["Seven"] = 7] = "Seven";
    Numbers[Numbers["Eight"] = 8] = "Eight";
    Numbers[Numbers["Nine"] = 9] = "Nine";
})(Numbers || (exports.Numbers = Numbers = {}));
var Teens;
(function (Teens) {
    Teens[Teens["Ten"] = 10] = "Ten";
    Teens[Teens["Eleven"] = 11] = "Eleven";
    Teens[Teens["Twelve"] = 12] = "Twelve";
    Teens[Teens["Thirteen"] = 13] = "Thirteen";
    Teens[Teens["Fourteen"] = 14] = "Fourteen";
    Teens[Teens["Fifteen"] = 15] = "Fifteen";
    Teens[Teens["Sixteen"] = 16] = "Sixteen";
    Teens[Teens["Seventeen"] = 17] = "Seventeen";
    Teens[Teens["Eighteen"] = 18] = "Eighteen";
    Teens[Teens["Nineteen"] = 19] = "Nineteen";
})(Teens || (exports.Teens = Teens = {}));
var Multiples;
(function (Multiples) {
    Multiples[Multiples["Twenty"] = 20] = "Twenty";
    Multiples[Multiples["Thirty"] = 30] = "Thirty";
    Multiples[Multiples["Forty"] = 40] = "Forty";
    Multiples[Multiples["Fifty"] = 50] = "Fifty";
    Multiples[Multiples["Sixty"] = 60] = "Sixty";
    Multiples[Multiples["Seventy"] = 70] = "Seventy";
    Multiples[Multiples["Eighty"] = 80] = "Eighty";
    Multiples[Multiples["Ninety"] = 90] = "Ninety";
})(Multiples || (exports.Multiples = Multiples = {}));
var Magnitudes;
(function (Magnitudes) {
    Magnitudes[Magnitudes["Hundred"] = 100] = "Hundred";
    Magnitudes[Magnitudes["Thousand"] = 1000] = "Thousand";
    Magnitudes[Magnitudes["Million"] = 1000000] = "Million";
    Magnitudes[Magnitudes["Billion"] = 1000000000] = "Billion";
    Magnitudes[Magnitudes["Trillion"] = 1000000000000] = "Trillion";
    Magnitudes[Magnitudes["Quadrillion"] = 1000000000000000] = "Quadrillion";
})(Magnitudes || (exports.Magnitudes = Magnitudes = {}));
